for_devs/local-net

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env /usr/local/bin/python3
```

Comment:
this should not have been changed probably ?

Reply:
We want to use the python version from `venv` so this is incorrect

Reply:
Sorry, missed that. That should be only my local change